pontoon/base/models.py

```diff
@@ -330,12 +330,14 @@ class Repository(models.Model):
         appended to the end of their path so that they are detected as
         source directories.
     """)
-    multi_locale = models.BooleanField(default=False, help_text="""
-        If true, this repo corresponds to multiple locale-specific repos. The
-        URL should have the string "{locale_code}" in it, which will be replaced
-        by the locale codes of all enabled locales for the project during pulls
-        and and commits.
-    """)
+
+    @property
+    def multi_locale(self):
+        """
+        Checks if url contains locale code variable. System will replace this variable by the locale codes
```

Comment:
I like to wrap comments at 72 characters per line as per [PEP8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length) for readability purposes. I suggest you do that here too.

Also, you should be explicit about what the locale code variable is. Just straight up putting `{locale_code}` in the docstring is fine.